src/setup/setup_framework.py

```diff
@@ -2,11 +2,22 @@
 import setup.argparser as argparser
 import setup.interactor as interactor
 
+# TODO: It is necessary to implement a config.json or a parameters.json for 
```

Comment:
It is necessary to implement a config.json or a parameters.json for hyperparameters about the search-engine.

Reply:
Addressed in #35